common/src/main/java/org/l2x6/cq/common/CqCatalog.java

```diff
@@ -82,6 +83,17 @@ public String getArtifactId() {
         }
     }
 
+    static final Predicate<Kind> SUPPORTED_CATALOG_KING_FILTER = kind -> {
```

Comment:
```suggestion
    static final Predicate<Kind> SUPPORTED_CATALOG_KIND_FILTER = kind -> {
```

Catalog is the king!

Reply:
Heh - nice catch 🤴 

I'll fix it.